packages/codemods/src/codemods/v0.39.x/updateBabelConfig/__tests__/updateBabelConfig.test.ts

```diff
@@ -0,0 +1,19 @@
+import transform from '../updateBabelConfig'
+test('Removes babel config for default setup', async () => {
+  await matchFolderTransform(transform, 'default')
+})
+
+test('Should preserve custom web config', async () => {
+  // Expecting:
+  // a) .babelrc -> babel.config.js
+  // b) removes "extends" property from config
+  await matchFolderTransform(transform, 'webCustom')
+})
+
+test('Should throw if root babel.config.js has custom config', async () => {
+  expect(async () =>
+    matchFolderTransform(transform, 'rootCustom')
+  ).rejects.toThrowError(
+    'Cannot automatically codemod your project. Please move your root babel.config.js settings manually'
+  )
+})
```

Comment:
Do we need to wrap this in a function?
```suggestion
test('Should throw if root babel.config.js has custom config', async () => {
  expect(matchFolderTransform(transform, 'rootCustom')).rejects.toThrowError(
    'Cannot automatically codemod your project. Please move your root babel.config.js settings manually'
  )
})
```

Reply:
Yeah I believe we do, the syntax is a bit strange. But expect with rejects will actually call the function. In order to pass in the params we need to wrap it in a function